pages/instructorQuestionSettings/instructorQuestionSettings.ejs

```diff
@@ -111,7 +135,7 @@
                ((authz_data.has_course_permission_edit) && (! course.example_course))) { %>
         <div class="card-footer">
           <div class="row">
-            <% if (authz_data.editable_courses && (authz_data.editable_courses.length > 0) && authz_data.has_course_permission_view) { %>
+            <% if (authz_data.editable_courses && (authz_data.editable_courses.length > 0) && authz_data.has_course_permission_view && question.course_id === course.id) { %>
```

Comment:
Should we use this same check elsewhere on the page too? In general, I'm hesitant to reuse the same page for shared questions. If we do, we should triple-check all the data shown (e.g. ensure the "assessments" list doesn't include assessments from the source course, hide the "statistics" tab or filter it to the target course, etc.)

Reply:
Right ya, hiding that other stuff was still on my list to do. 

I can also check to see if it makes sense to actually just make a different page instead of having if statements everywhere.